cmd/app.go

```diff
@@ -1080,6 +1083,80 @@ func createNamespace(c *cliclient.MasterClient, n string) error {
 	return nil
 }
 
+func processFileInputs(
+	ctx *cli.Context,
+	c *cliclient.MasterClient,
+	tv *managementClient.TemplateVersion,
+	answers map[string]string,
+	values string,
+	interactive bool) (map[string]string, string, error) {
+	var err error
+	if ctx.String("answers") != "" && ctx.String("values") != "" {
+		return nil, "", fmt.Errorf("cannot process an answers file and a values file")
+	}
+	if hasAnswers(ctx.String("answers"), answers) || !hasValues(ctx.String("values"), values) {
+		values = ""
+		answers, err = processAnswers(ctx, c, tv, answers, interactive)
+		if err != nil {
+			return answers, values, err
+		}
+	}
+	if hasValues(ctx.String("values"), values) {
+		answers = nil
```

Comment:
What will happen here if a user does have both set on the app and we go to do an upgrade?

Reply:
it will hit the hasAnswers block first so it will keep and update the answers and empty the values, unless they are passing in a values file.

Reply:
If both are set I don't think we should blank out anything.

Reply:
added support for both values to be kept unless new files are passed in